core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -58,7 +58,7 @@
 public abstract class AbstractTomcatContainer implements TomcatContainer {
 
   /** The logger. */
-  protected Log logger = LogFactory.getLog(getClass());
+  protected Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
I prefer the LogFactory.getLog(AbstractTomcatContainer.class) way. I did not change this to keep the system default usage.